featurebyte/service/feature_list.py

```diff
@@ -393,3 +393,17 @@ async def get_feature_list_info(self, document_id: ObjectId, verbose: bool) -> F
             versions_info=versions_info,
             deployed=feature_list.deployed,
         )
+
+
+class AllFeatureListService(
+    BaseDocumentService[FeatureListModel, FeatureListServiceCreate, FeatureListServiceUpdate]
```

Comment:
Can this inherit from `FeatureListService` like the other `All*Service` or is there a reason not to?

Reply:
FeatureListService depends on a number of catalog-specific services, if this inherits from that then those services cannot be initialized without a catalog_id